src/core/lib/iomgr/tcp_posix.cc

```diff
@@ -287,7 +301,7 @@ static void finish_estimate(grpc_tcp* tcp) {
   /* If we read >80% of the target buffer in one read loop, increase the size
      of the target buffer to either the amount read, or twice its previous
      value */
-  if (tcp->bytes_read_this_round > tcp->target_length * 0.8) {
+  if (tcp->bytes_read_this_round >= tcp->target_length) {
```

Comment:
are these adjustments based on experiments?

Reply:
It's not strictly needed. It was needed for an earlier version of this comment. I'll revert this one.

Reply:
I'm not against these adjustments, It's just it'd be nice to have some justification in the documentation.